### PR TITLE
New feature: Update the style of the admin file manager

### PR DESCRIPTION
**Pull request by jack-han-115**
Fixed issue #<Mantis issue number>:
New feature #<Mantis issue number>:
Dev:

### CODE REVIEW

File: assets/packages/adminbasics/src/adminbasicsmain.js [hunk 1]
```diff
@@ -175,4 +180,4 @@ const AdminCore = function(){
         onLoadRegister();
 }
 
-AdminCore();
+AdminCore();
```
Comment (olleharstedt) [inline assets/packages/adminbasics/src/adminbasicsmain.js:183]: Not sure I understand why this change is needed for file manager?

File: assets/packages/adminbasics/src/components/file-manager.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    {
+        originIcon: "a[href='kcact:mvdir'] span",
+        newIcon: "ri-eraser-line",
+    },
+    {
+        originIcon: "a[href='kcact:rmdir'] span",
+        newIcon: "ri-delete-bin-line",
+    },
+
+    // folder regular
```
Comment (olleharstedt) [inline assets/packages/adminbasics/src/components/file-manager.js:59]: Hm what does this mean? :)

File: assets/packages/adminbasics/src/components/file-manager.js [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+            // Load sea_green css again after iframe is fully loaded
+            handleAppendCssLink({
+                header: fileManagerIframe.contentWindow.document.head,
+                linkUrl: "/themes/admin/Sea_Green/css/sea_green.css",
+            });
+            handleAppendCssLink({
+                header: fileManagerIframe.contentWindow.document.head,
+                linkUrl:
+                    "http://ls-ce/assets/fonts/font-src/remix/remixicon.css",
```
Comment (olleharstedt) [inline assets/packages/adminbasics/src/components/file-manager.js:197]: Hardcoded link to what?

File: assets/packages/adminbasics/src/components/file-manager.js [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+    },
+    {
+        originIcon: "a[href='kcact:rmdir'] span",
+        newIcon: "ri-delete-bin-line",
+    },
+
+    // folder regular
+];
+
+const handleReplaceIcons = ({ iframeSource, originIcon, newIcon }) => {
```
Comment (olleharstedt) [inline assets/packages/adminbasics/src/components/file-manager.js:62]: All these functions are missing docblocks. Is that by design? What's our current policy?

File: assets/packages/adminbasics/src/adminbasicsmain.js (hunk 1 above)
Comment (jack-han-115) [reply assets/packages/adminbasics/src/adminbasicsmain.js:183]: we are loading fileManager in the iframe.
the custom styles are loaded first and the iframe is loaded after that.
so styles are not applied.
so, i added file-manager.js in the adminbasicsmain.js and load seagreen css and remix icons style after the iframe is loaded fully.

File: assets/packages/adminbasics/src/components/file-manager.js (hunk 3 above)
Comment (olleharstedt) [reply assets/packages/adminbasics/src/components/file-manager.js:219]: Resolved but still hardcoded? :) What's the intended behaviour?

File: assets/admin_themes/Sea_Green/fileManager/fileManager.scss [hunk 5]
```diff
@@ -0,0 +1,308 @@
...
+      display: none;
+    }
+    .about div.head a {
+      background: none;
+    }
+  }
+
+  #menu {
+    .ui-menu {
+      box-shadow: none !important;
```
Comment (olleharstedt) [inline assets/admin_themes/Sea_Green/fileManager/fileManager.scss:70]: Lots of `important` going on. Is that needed even when the selector is made more precise than the previous "default" CSS selectors? Dunno which is best.
